product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -292,7 +290,7 @@ After you create the cluster in the BigAnimal console, the UI will display the *
     <details><summary>GCP</summary>
 
     1.  Copy the **service account** to your clipboard.
-    1.  Go to the Google Cloud console, select **Security**, **VIEW BY PRINCIPALS**, **GRANT ACCESS** for your key.
+    1.  On the Google Cloud console, select **Security**, **VIEW BY PRINCIPALS**, and **GRANT ACCESS** for your key.
```

Comment:
I wasn't sure of this edit. It was a bit difficult to follow this step. Are these three options to select? It's unclear to me as a reader what this meant.

Reply:
Yes, it is three options depending on the cloud provider for your key. Let me add a suggestion to make this clearer at the top of this step.